test/cypress/pages/owning_a_home.js

```diff
@@ -0,0 +1,44 @@
+export class ExploreRates {
+    constructor() {}
+
+    open() {
+        cy.visit('/owning-a-home/explore-rates/')
+    }
+
+    selectState(state) {
+        cy.get('#location')
+          .select(state)
+    }
+    
+    graph() {
+      return cy.get('#chart-section')
+        .within(() => {
+          return cy.get('figure:first');
+        });
+    }
+
+}
+
+export class Feedback {
+  constructor() {}
+
+  open() {
+    cy.visit('/owning-a-home/feedback');
+  }
+
+  submitComment(comment) {
+    cy.get('#comment').type(comment);
+    cy.get('.content_main')
+      .within(() => {
+        cy.get('form').submit();
```

Comment:
You should prob stub this XHR route to block form submissions (a la how you did `stubSubscriptionResponse` above). Something like:

```
stubFeedbackResponse() {
    cy.server()
    cy.route({
      method: 'POST',      
      url: '/owning-a-home/feedback/',
      response: '{"result": "pass", "heading": "", "message": "Thanks for your feedback!"}'
    });
  }
```

Reply:
The stubbed subscription response for the email sign-up form is to get around the fact that we don't currently set any GovDelivery env vars in the dev Docker stacks. To truly test the Python that backs those forms, we need to do that to actually send a request to the GovDelivery staging environment.

Here, because again the goal is to test that the Python machinery is still working, we want to actually post the form and record it in the DB.

Reply:
Sounds good 👍